dcolor.py

```diff
@@ -24,19 +24,19 @@ def makeDomain(self):
 
     def makeColorModel(self, zz):
         """Create the HSV color model for the function domain that will be plotted"""
-        H = self.normalize(np.mod(np.angle(zz),2*np.pi)) #Hue determined by arg(z)
-        r = np.log2(np.add(1.0,np.abs(zz)))
-        S = np.divide(np.add(1.0,np.abs(np.sin(np.multiply(2.0*np.pi,r)))),2.0)
-        V = np.divide(np.add(1.0,np.abs(np.cos(np.multiply(2*np.pi,r)))),2.0)
+        H = self.normalize(np.angle(zz) % (2*np.pi)) #Hue determined by arg(z)
+        r = np.log2(1. + np.abs(zz))
+        S = (1. + np.abs(np.sin(2.*np.pi * r))) / 2.
```

Comment:
should we be consistent with spaces between operators? i.e. `2. * np.pi`

Reply:
Yeah clearly! gonna fix this !